content/_getting-started/first-flow.md

```diff
@@ -1,172 +1,123 @@
+We want to use Simple Trigger, so we will start typing "simple" to the search bar, and find it faster. Then select it and click "Choose Simple trigger":
 
-![Select the trigger](/assets/img/getting-started/first-flow/getting-started-flow-004.png "Select the trigger")
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_6.png)
 
-Typically a component can have more than one triggers to retrieve different
-objects from the source API. Here we use the Petstore component to learn the
-{{site.data.tenant.name}} platform. It provides 3 different triggers to retrieve
-pets by their status. These 3 triggers differ by the implemented code style. So
-select any of them, for example click on the *Get Pets By Status With Promises*
-radio button to continue.
+The next step is [data sample](/data-sample-overview) retrieval. Click "Retrieve sample from Simple trigger" and then "Continue" to proceed:
 
-![Choose trigger parameter](/assets/img/getting-started/first-flow/getting-started-flow-005.png "Choose trigger parameter")
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_7.png)
 
-The *Get Pets By Status With Promises* trigger returns `Available`, `Pending` and
-`Sold` values for the `Pet Status` parameter. Click on drop-down menu to open it.
-Select the `Available` option and click on *Continue* button.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_8.png)
 
-![Retrieve the sample](/assets/img/getting-started/first-flow/getting-started-flow-006.png "Retrieve the sample")
+In the next tab you may add a description if you want, and click "Finish step" to finalize the initial trigger:
 
-Now we retrieve a sample from Petstore API where the pet status is `Available`.
-When the process is successful you will see the following page to choose retrieved sample.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_9.png)
 
-![Choose this sample](/assets/img/getting-started/first-flow/getting-started-flow-007.png "Choose this sample")
 
-Click to *Choose this sample* and continue. Next you will see the Retrieve sample page again.
+## Configuring Action Steps
 
-![Continue to finish step1 configuration](/assets/img/getting-started/first-flow/getting-started-flow-008.png "Continue to finish step1 configuration")
+As our initial trigger is ready, let's proceed with the second step, which is an action triggered by the first step. To start configuring the action, click "Add a new action":
 
-Please click on Continue button here to go further.
+![](/assets/img/getting-started/creating-basic-flow/Screenshot_10.png)
 
-> **Note**: Please don't click on *Retrieve sample from Petstore API (Node.js)*
-> button again. You will get back to the earlier stage to select the sample again.
-> This is not a problem, but unnecessary at this stage.
+Again, we're prompted to setup the step, similar to the first one. We have chosen Node.js as the second Component. Select it and click "Choose Node.js Code":
```

Comment:
Phew! Node.js in the article about a Basic integration Flow? We assume the readers of the platform are beginners and Node.js is way to heavy for them. Let's choose another action